### ISSUE TITLE
Notebooks utilizing ipywidgets.Output() don't work with progressive rendering

### ISSUE THREAD

**Issue by timkpaine** [voila-dashboards/voila#394]
include the following in a cell after the first

```python3
class Test(object):
    def __init__(self):
        self.view = ipywidgets.Output()
t = Test()
```

**Comment by martinRenou** [comment]
I am not sure I understand this issue... What is not working exactly?

**Comment by timkpaine** [comment]
The notebook won't render at all, there's more background on gitter

**Comment by timkpaine** [comment]
#396 might inadvertently fix this

**Comment by martinRenou** [comment]
Ok, sorry I did not pay attention to the gitter discussion. I will try on my branch and on master.

**Comment by martinRenou** [comment]
You are talking about this notebook, right?
```python
# Cell 1
import time
from ipywidgets import *
from IPython.display import display

# Cell 2
class Test(object):
    def __init__(self):
        self.layout = Output()
        with self.layout:
            print('Hello world')
        
    def render(self):
        display(self.layout)

# Cell 3
time.sleep(10)
t = Test()

# Cell 4
t.render()
```

I added the `print('Hello world')` so that we have something displayed. It seems to work for me both on master and on #396... Might be a race condition

**Comment by timkpaine** [comment]
What happens if you don't use the output, just instantiate it e.g. remove the `with layout`

**Comment by martinRenou** [comment]
Well it does not show anything as expected. The `Output` is there, just not visible because it's empty.
You could see it if you change its style like `self.layout.layout.border = '2px dashed red'`, you would see it's there, just empty

**Comment by timkpaine** [comment]
For me the page spins, and nbconvert hangs on connecting to the existing kernel

**Comment by martinRenou** [comment]
Weird. I cannot reproduce it locally.

The thing is the current master reconnects to the kernel for each cell... That is fixed in 396, which may be why your use case now works.

**Comment by timkpaine** [comment]
Yeah I will test to make sure. It was hanging on a `wait_for_kernel` but since we don't disconnect now it should be fine
